python/ml_croissant/ml_croissant/_src/structure_graph/nodes/source.py

```diff
@@ -103,11 +106,13 @@ def from_jsonld(cls, issues: Issues, jsonld: Json) -> list[Transform]:
                 )
                 continue
             format = transform.get(constants.ML_COMMONS_FORMAT)
+            json_path = transform.get(constants.ML_COMMONS_JSON_PATH)
             regex = transform.get(constants.ML_COMMONS_REGEX)
             replace = transform.get(constants.ML_COMMONS_REPLACE)
             separator = transform.get(constants.ML_COMMONS_SEPARATOR)
             if (
                 format is None
```

Comment:
if we expect this to grow we could also `if all(v is None for v in [format, json_path, ...])``

Reply:
Done.